Makefile

```diff
@@ -274,6 +275,8 @@ check: copyright-check lint shell-check
 clean-test-results:
 	@rm -f test.log
 	@go clean -testcache
+	@rm -f $(COVER_ROOT)/*
+	@rm -f $(REPORT_ROOT)/*
```

Comment:
I tested this and it works just fine when the folder isn't there.

> By default, the rm command returns an error message if the file being deleted doesn't exist. But, we can use the -f option to prevent prompts, ignore missing files, and prevent the rm command from returning an error message even when files are missing

Reply:
sorry, I updated my comment a minute after I sent it

Reply:
I merged the 3 `rm` calls now. Just without `-r` as I get a little nervous whenever that's present 😅 still does the job!